v0/util/index.js

```diff
@@ -1151,6 +1151,43 @@ function isCdkDestination(event) {
   );
 }
 
+/**
+ * This function helps to remove any undefined value from dynamicForm, dynamicCustomForm
+ * and dynamicSelectForm array.
+ *
+ * example:
+ * input: "eventsToEvents": [
+                    {
+                        "from": "spin_result",
+                        "to": "Schedule"
+                    },
+                    {
+                        "to": "Schedule"
+                    }
+                ]
+ * output: "eventsToEvents": [
+                    {
+                        "from": "spin_result",
+                        "to": "Schedule"
+                    }
+                ]
+
+ * @param {*} eventArray -> This is the dynamicForm/dynamicCustomForm/dynamicSelectForm element.
+ * @param {*} keyLeft -> This is the name of the leftKey, in general it is 'from'.
+ * @param {*} keyRight -> This is the name of the rightKey, in general it is 'to'.
+ * @returns
+ */
+function validateDynamicFormAttribute(attributeArray, keyLeft, keyRight) {
```

Comment:
It'd be more appropriate.
```suggestion
function validateDynamicFormConfig(attributeArray, keyLeft, keyRight) {
```